cmd/buildah/images.go

```diff
@@ -245,9 +251,9 @@ func outputHeader(opts imageOptions) string {
 	if opts.quiet {
 		return formats.IDString
 	}
-	format := "table {{.Name}}\t{{.Tag}}\t"
+	format := "table {{if .Name}}{{.Name}}{{else}}<none>{{end}}\t{{if .Tag}}{{.Tag}}{{else}}<none>{{end}}\t"
 	if opts.noHeading {
-		format = "{{.Name}}\t{{.Tag}}\t"
+		format = "{{if .Name}}{{.Name}}{{else}}<none>{{end}}\t{{if .Tag}}{{.Tag}}{{else}}<none>{{end}}\t"
```

Comment:
(Absolutely non-blocking nit: Is the difference the initial `"table "`? Maybe do a
```go
format := ""
if !opts.noHeading {
    format += "table "
}
format += {{…}}`
```
to be clear about the difference, and to structurally ensure that the two column definitions are the same.)

Reply:
Yes, it is.  Changing it.